IBAnimatable/Animatable.swift

```diff
@@ -44,9 +44,14 @@ public protocol Animatable: class {
    */
   var force: CGFloat { get set }
 
+  /**
+   Animation function as a timing curve. (default value should be none)
+   */
+  var timingFunction: TimingFunctionType { get set }
+
 }
 
-public extension Animatable where Self: UIView {
+public extension Animatable {
```

Comment:
Forgot to remove this change for this PR
(Trying to make `UIBarItem` also `Animatable`, I see that I can remove that)